assembly/std/storage.ts

```diff
@@ -239,14 +239,21 @@ export function get<T>(key: T): T {
 }
 
 /**
- * Returns (key, value) in the datastore of the given address.
+ * Returns the value associated with the given `key` in the datastore of the contract at the specified `address`.
  *
- * @privateRemarks
- * TODO: explains what happens on missing key.
+ * @remarks
+ * If there is no value associated with the `key`, the function will throw an error.
+ * If the contract at the given address does not exist, the function will throw an error.
```

Comment:
Those two remarks should be using @throws

Reply:
We should also add in the module introduction a note explaining that thrown exception are uncachable and can be found in he event